src/methods/send_document.rs

```diff
@@ -67,6 +69,16 @@ impl<'a> SendDocument<'a> {
         self.reply_markup = Some(markup.into());
         self
     }
+
+    /// Configures automatic server-side content type detection.
+    /// Reflects the `disable_content_type_detection` parameter.
+    pub const fn should_disable_content_type_detection(
+        mut self,
+        should_disable: bool,
+    ) -> Self {
+        self.disable_content_type_detection = Some(should_disable);
+        self
+    }
```

Comment:
`SendDocument` is supposed to inherit this field from `self.document`. Also note that this field needs to be serialized in `call`.